packages/medusa/src/types/global.ts

```diff
@@ -114,6 +115,9 @@ export type ConfigModule = {
   projectConfig: {
     redis_url?: string
 
+    // use to configure global options passed to `EventBusService.emit` i.e. the options passed to Bull
+    global_job_options?: Record<string, unknown> & EmitOptions
```

Comment:
thought: wouldn't this have to be deprecated as soon as we move to event bus modules? If so, wouldn't it be a bad idea to introduce that now?

Reply:
That is true, it will have to move to the module has well. But for the users still user version below 1.8 it makes sense to keep it configurable here for them I think, wdyt?